libs/entities/src/types/login.ts

```diff
@@ -1,10 +1,16 @@
-import { Users } from '@prisma/client';
+
 import { TToken } from './token';
 
 export type TLoginResponse = {
   id: string;
   token: TToken;
-  user: Users;
+  user: {
+    nik: string;
+    fullname: string | null;
```

Comment:
ieu mun erek optional mah fullname?: string we gak usah pake | null lagi

Reply:
noted puh